examples/with-jest/testSetup.js

```diff
@@ -0,0 +1,4 @@
+import {matcher, serializer} from 'jest-glamor-react'
+
+expect.addSnapshotSerializer(serializer)
+expect.extend(matcher)
```

Comment:
Could you add a comment above this line that says:

> // this adds `toMatchSnapshotWithGlamor` to `expect` and makes the snapshot diff output look nice in the terminal

I realize that this may be more in the scope of the `jest-glamor-react` docs, but I think this could still be helpful :) Thanks!